imdown-mobile/events/event_detailed_styles.tsx

```diff
@@ -0,0 +1,111 @@
+import { StyleSheet } from "react-native";
+export const EventDetailedStyles = StyleSheet.create({
```

Comment:
Don't have to do this now but something to think about is if we could move some of these values into shared constants

Reply:
sure. i'll address this in a follow-up PR